examples/text/document_classification_20newsgroups.py

```diff
@@ -83,7 +83,12 @@
               help="Remove newsgroup information that is easily overfit: "
                    "headers, signatures, and quoting.")
 
-(opts, args) = op.parse_args()
+
+def is_interactive():
+    return not hasattr(sys.modules['__main__ '], '__file__')
+
+argv = [] if is_interactive() else sys.argv[1:]
```

Comment:
Considering that the user running the example won't be encountering the error anymore, do you feel a comment is necessary? I felt that it could lead to some kind of ambiguity as to why was the workaround added. Please let me know what you feel.